.github/workflows/release-phar.yml

```diff
@@ -0,0 +1,43 @@
+on:
+  release:
+    types:
+      - created
+
+name: Attach PHAR to release
+
+jobs:
+  build:
+    name: Compile and upload Phar
+    runs-on: ubuntu-18.04
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v2
+        with:
+            fetch-depth: 0
+
+      - name: Set PHP 7.3
+        uses: shivammathur/setup-php@v2
+        with:
+          php-version: '7.3'
```

Comment:
I guess still up for discussion on whether to upgrade minimum version, but can change this along with whatever changes we do for the min version bump

Reply:
yeah